terraform-tests/mssql_test.go

```diff
@@ -85,6 +85,11 @@ func testTerraformMssql(region string) {
 
 	validVPC := awsVPCID
 
+	// The following two resources were manually created
+	// in the AWS console to be able to test some corner-cases
+	vpcWithMoreThan20Subnets := "vpc-066fae2eced5df2fb"
```

Comment:
Can we make these `const`s? That way the compiler will fail if there's any attempt to modify them. I've seen some tricky test flakes that were causes by `var`s being accidentally modified in one test, causing subsequent tests to fail.